inst/extdata/variable_details.csv

```diff
@@ -111,19 +111,19 @@ ADL_07,ADL_07_cat2_2,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2001_i, cchs20
+ADL_score_5,ADL_score_5_cat6_5,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p, cchs2015_2016_p, cchs2017_2018_p, cchs2009_s, cchs2010_s, cchs2012_s, chs2001_i, cchs2003_i, cchs2005_i, cchs2007_2008_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i","DerivedVar::[ADL_01, ADL_02, ADL_03, ADL_04, ADL_05]",,N/A,4,6,Needs help with at least 4 tasks,Needs help with at least 4 tasks,N/A,N/A,Needs help with at least 4 tasks,ADL score,"Derived using the ADL variables common to all cycles from 2001 to 2014 (ADL_01,  ADL_02,  ADL_03,  ADL_04, ADL_05) to represent the number of tasks that an individual needs help with.",,,,active,,No typeStart
+ADL_score_5,ADL_score_5_cat6_6,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p, cchs2015_2016_p, cchs2017_2018_p, cchs2009_s, cchs2010_s, cchs2012_s, chs2001_i, cchs2003_i, cchs2005_i, cchs2007_2008_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i","DerivedVar::[ADL_01, ADL_02, ADL_03, ADL_04, ADL_05]",,N/A,5,6,Needs help with at least 5 tasks,Needs help with at least 5 tasks,N/A,N/A,Needs help with at least 5 tasks,ADL score,"Derived using the ADL variables common to all cycles from 2001 to 2014 (ADL_01,  ADL_02,  ADL_03,  ADL_04, ADL_05) to represent the number of tasks that an individual needs help with.",,,,active,,No typeStart
+ADL_score_5,ADL_score_5_cat6_NA::a,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p, cchs2015_2016_p, cchs2017_2018_p, cchs2009_s, cchs2010_s, cchs2012_s, chs2001_i, cchs2003_i, cchs2005_i, cchs2007_2008_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i","DerivedVar::[ADL_01, ADL_02, ADL_03, ADL_04, ADL_05]",,N/A,NA::a,6,not applicable,not applicable,N/A,N/A,not applicable,ADL score,"Derived using the ADL variables common to all cycles from 2001 to 2014 (ADL_01,  ADL_02,  ADL_03,  ADL_04, ADL_05) to represent the number of tasks that an individual needs help with.",,,,active,,No typeStart
+ADL_score_5,ADL_score_5_cat6_NA::b,cat,"cchs2001_p, cchs2003_p, cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cchs2010_p, cchs2011_2012_p, cchs2012_p, cchs2013_2014_p, cchs2014_p, cchs2015_2016_p, cchs2017_2018_p, cchs2009_s, cchs2010_s, cchs2012_s, chs2001_i, cchs2003_i, cchs2005_i, cchs2007_2008_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i","DerivedVar::[ADL_01, ADL_02, ADL_03, ADL_04, ADL_05]",,N/A,NA::b,6,missing,missing,N/A,N/A,missing,ADL score,"Derived using the ADL variables common to all cycles from 2001 to 2014 (ADL_01,  ADL_02,  ADL_03,  ADL_04, ADL_05) to represent the number of tasks that an individual needs help with.",,,,active,,No typeStart

```

Comment:
Same as [this comment](https://github.com/Big-Life-Lab/cchsflow/pull/140/commits/db122101b4616be897bba7504d1a33352759da97#r2291391462).